src/binder/bind/bind_standalone_call.cpp

```diff
@@ -24,6 +24,11 @@ std::unique_ptr<BoundStatement> Binder::bindStandaloneCall(const parser::Stateme
     }
     auto optionValue = expressionBinder.bindExpression(*callStatement.getOptionValue());
     ExpressionUtil::validateExpressionType(*optionValue, ExpressionType::LITERAL);
+    if (LogicalTypeUtils::isFloatingPoint(optionValue->dataType) &&
+        LogicalTypeUtils::isInteger(LogicalType(option->parameterType))) {
+        throw BinderException{
+            "Invalid cast from Double to Int during: " + callStatement.getOptionName() + "."};
```

Comment:
The source type might be float, you can use LogicalTypeUtils::toString(optionValue->dataType) to convert the type to string.
Also the int can be int32/int64, you can also use `LogicalTypeUtils::toString(optionValue->parameterType)`